mas/mints/mapping.txt

```diff
@@ -0,0 +1,6 @@
+token_id-"component_id"
+?-contract/zarathustra/grow_registry:0.1.0
+?-skill/zarathustra/plantation_abci/0.1.0
+?-skill/zarathustra/plantation_station_abci/0.1.0
+?-agent/zarathustra/plantation/0.1.0
+?-service/eightballer/plantation_station/0.1.0
```

Comment:
It will be important to know what the dependencies are of each package, including already existing dependencies in the registry. Atm you need to do this manually, simplest is to check for instance here: https://github.com/valory-xyz/open-autonomy/blob/main/mints/mapping.txt for the token id

Reply:
Should this be added to the mapping .txt yeah

Reply:
Temporarily yes! What I do is this: 
```
skill/zarathustra/plantation_abci/0.1.0 -> 14, ...
```
14 here related to valory/abstract_round_abci:0.1.0 which I get from either on-chain or the list here (https://github.com/valory-xyz/open-autonomy/blob/main/mints/mapping.txt)... do the same for all deps; you'll need that info during minting